sbt-bridge/src/xsbt/DelegatingReporter.java

```diff
@@ -86,7 +86,9 @@ public void doReport(MessageContainer cont, Context ctx) {
           return Optional.ofNullable(src.file().path());
```

Comment:
Would be clearer to use `if (src.exists())`, same for `sourceFile` above.

Reply:
Done